some updates on bult-in gameplay help

illwieckz opened the pull request:
some updates on bult-in gameplay help

slipher commented on ui/help_gameplay.rml:
```diff
@@ -37,26 +37,44 @@
+				leech. These buildings need to be spread out for efficiency.
+				<br />
+				<br />
+				Don't build mining structures too close together or they will compete with each others
```
s/others/other

illwieckz replied:
done

slipher commented on ui/help_gameplay.rml:
```diff
@@ -122,7 +122,7 @@
 				<img class="emoticon" src="/emoticons/advgranger" />
 				advanced granger's spit attack can be used to put out
-				fires.
+				fires or slow downs intruders!
```
s/downs/down/

illwieckz replied:
done

illwieckz commented:
I added more text to that gameplay guide.

slipher commented on ui/help_gameplay.rml:
```diff
@@ -23,40 +23,98 @@
+				enough energy to remain operational.
+				<br />
+				<br />
 				Building a structure requires build points, which are now a one-time
```
I think this is incorrect now... the BP pool is just determined by the mining structures, right?

illwieckz replied:
No, the mining structures continue to mines resources (with the rate decreasing over time). And in all case, the BP are lost.

Let's imagine I build a drill, let's say it enable 40 bp (I don't remember the exact number, but that's close) and mines 20 other bp. So I have 60 bp.

After I build 6 turrets which costs 10 bp each. So I spent all the bp.

4 turrets have been destroyed but the drill is still alive, I need to find 40bp again to rebuild those 2 turrets, for example by building a second drill.

Then the next 2 turrets have been destroyed as well, I need to find 20bp to rebuild those 2 turrets.

If I loose a drill, I will lose 40bp and I will miss 40bp, some buildable will disable themselves, rebuilding the drill would just make me miss no bp, I would gain no bp by building it.

That's why at some point, some team ends with -100 bp and they can't build anything even by spamming drills because they lost too much bp and they would never be able to build enough drills to recover, this replace sudden death.

slipher replied:
The build points are not permanently lost, but rather put into the "recovery queue". The BP exit the queue one at a time. The queue processes BP at an initial rate of 16/minute [g_BPRecoveryInitialRate]; this rate halves every 10 [g_BPRecoveryRateHalfLife] minutes.

illwieckz replied:
I'm not sure to understand everything, so I made this part of the help _more vague_, anyway there is enough information for the player to know how he can lose and earn build points.

slipher commented on ui/help_gameplay.rml:
```diff
@@ -131,45 +155,115 @@
+				<br/>
+				The
+				<img class="emoticon" src="/emoticons/booster" />
+				Booster is the fastest healing buildable, build one as soon
```
The wording here is not the clearest, how about "The booster gives rapid healing for nearby aliens"

illwieckz replied:
I applied the suggestion.

slipher commented on ui/help_gameplay.rml:
```diff
@@ -131,45 +155,115 @@
+				The new maximum of evos an alien can store is 20 (formerly 9).
+				<br />
+				<br />
+				Humans no longer have the Tesla defense tower neither the defense
```
s/neither/nor/

illwieckz replied:
done

slipher commented on ui/help_gameplay.rml:
```diff
@@ -131,45 +152,115 @@
 			</tab>
 			<panel>
-				The human dodge has been removed, stamina has been increased.
+				Since Tremulous GPP The human dodge has been removed,
```
Let's drop this paragraph since GPP is hardly what anyone will remember best nowadays.

illwieckz replied:
Was human dodge GPP specific (not in 1.1)?

slipher replied:
Yes

slipher commented on ui/help_gameplay.rml:
```diff
@@ -23,40 +23,95 @@
+				small pool of build points but if you want to place a greater number of
+				buildables in an area, you can build a
+				<img class="emoticon" src="/emoticons/leech" />
+				mining structure to increase the available power and extract more build points.
```
To me "extract" sounds like the build points would be stored permanently. I suggest "provide"

illwieckz commented:
I updated the PR. This PR does not aim to produce a perfect gameplay guide or this will never be ready, but at least fix some of the biggest issues of the former one that was not updated since probably five years or more. Like, if the problem is that Tremulous GPP is mentioned in the last chapter of the guide, it means the guide is better because previously that mention was the introduction and it was not only required for the player to remind GPP, but also remind all the details of GPP… So, I fixed that GPP mention but I hope you got it. Most remarks that can be done about the current state of the guide is not anymore about fixing the former guide, but about taking this guide as a reference to improve it as a next step. So it's for another PR anyone can submit once this one is merged.

illwieckz commented:
The various remarks has been addressed. Other changes are welcome in other PR.
